cribl_stream/README.md

```diff
@@ -0,0 +1,69 @@
+# Cribl Stream
+
+## Overview
+[Cribl Stream][1] helps you process machine data – logs, instrumentation data, application data, metrics, and so on – in real time, and deliver it to your analysis platform of choice. It allows you to:
```

Comment:
Can you please call out somewhere in the Overview section that this integration will be for on-prem customers only?

Reply:
Done, I added it to the readme.md under the Setup section.